Modix.Services/Promotions/PromotionsService.cs

```diff
@@ -505,10 +540,11 @@ private async Task PerformCommonCreateCampaignValidationsAsync(IGuildUser subjec
                 throw new InvalidOperationException($"An active campaign already exists for {subject.GetFullUsername()} to be promoted to {targetRankRole.Name}");
 
             // JoinedAt is null, when it cannot be obtained
+            /*
```

Comment:
Wat up wit dis?

Reply:
thats from when i was testing, it get left in in the initial. its already been removed